### DISCUSSION TITLE
Develco (Frient) HESZB-120 test mode

### DISCUSSION THREAD

**Discussion by Theo-NL** [Koenkk/zigbee2mqtt#8169] [Devices]
Hello,

I have two Frient HESZB-120 heat detectors installed (recognized and installed as Develco HESZB-120), both work within Domoticz and the plugin. 
Only the test mode doesn't work, in the Domoticz log appears always the message " 2021-07-28 23:14:03.494 Error: Zigbee2MQTT: (Zigbee2MQTT) Brandmelder keuken: can not process binary item "test"
2021-07-28 23:14:03.496 Error: Zigbee2MQTT: (Zigbee2MQTT) Brandmelder schuur: can not process binary item "test" 

I updated Zigbee2mqtt and the plugin.

What could be the problem ?

Thanks in advance !

Regards, Theo (NL).

**Comment by Koenkk** [comment]
Please create an issue here: https://github.com/stas-demydiuk/domoticz-zigbee2mqtt-plugin/issues , this is not directly a z2m issue.

**Reply by Theo-NL** [reply]
Ok, thanks, I'll do.
